### PR TITLE
content/docs: Add security section

### PR DESCRIPTION
**Pull request by vladandrew**
Signed-off-by: Vlad Badoiu <vlad_andrei.badoiu@upb.ro>

### CODE REVIEW

File: content/en/docs/features/security.md [hunk 1]
```diff
@@ -2,8 +2,26 @@
...
+commonly found in standard OSes (e.g., stack and page pro- tection, ASLR,
+etc.); Unikraft already supports several of these including CFI, UBSAN,
+Stack protector, KASAN, as well as initial support for hardware
+compartmentalization with Intel MPK and EPT as show by
+[FlexOS](https://project-flexos.github.io/). It should therefore be
+possible to achieve good security while retaining high performance with
+Unikraft.
 
+Further, Unikraft supports memory safe programming languages such as Rust
+for writing both internal and external libraries. From a cryptographic
```
Comment (razvand) [inline content/en/docs/features/security.md:26]: Is the cryptographic support tied to Rust? If not, it should be using another paragraph.

File: content/en/docs/features/security.md [hunk 2]
```diff
@@ -2,8 +2,26 @@
 title: Security
 date: 2020-01-11T14:09:21+09:00
 draft: false
+collapsible: false
 weight: 301
 ---
 
 ## Unikraft Security
+ 
+Unikernels traditionally have had serious security issues, but we argue
```
Comment (razvand) [inline content/en/docs/features/security.md:11]: Is this true? Why do we say "traditionally have had serious security issues"?

File: content/en/docs/features/security.md [hunk 3]
```diff
@@ -2,8 +2,26 @@
...
+that those have been implementation artifacts rather than fundamental
+problems. In fact, unikernels are used commercially in security-minded
+domains such as automotive because their high level of specialization means
+that they provide a small Trusted Computing Base. Having said that, past
+unikernel projects have failed to provide standard security features
```
Comment (razvand) [inline content/en/docs/features/security.md:16]: This seems a bit strong. Is it true for all / most unikernel projects?

**Review by razvand — COMMENTED** [review]
I added my comments.

File: content/en/docs/features/security.md (hunk 2 above)
Comment (vladandrew) [reply content/en/docs/features/security.md:11]: There is this paper, Assessing Unikernel Security by NCC Group that basically confirms this.

File: content/en/docs/features/security.md (hunk 3 above)
Comment (vladandrew) [reply content/en/docs/features/security.md:16]: Yes, the same paper from NCC tackles this.

**Review by razvand — CHANGES_REQUESTED** [review]
I added my comments.

**Comment by nderjung** [comment]
Closing in favour of #51.

**Pull request closed without merging**